browser/src/VariantPage/VariantSiteQualityMetrics.tsx

```diff
@@ -467,7 +470,8 @@ const getAvailableMetrics = (datasetId: any) => {
   if (isV4(datasetId)) {
     return [
       'SiteQuality',
-      'InbreedingCoeff',
+      'inbreeding_coeff',
+      // 'InbreedingCoeff',
```

Comment:
Is this commented out metric needed for some reason? If not, it should be removed.

Reply:
Also, it's a bit odd that when selecting the metrics, this metric is the only one in lower case. 

![Screenshot 2023-11-07 at 14 22 10](https://github.com/broadinstitute/gnomad-browser/assets/59549713/236de9c9-927f-4b84-b260-50544b5c51ec)

Reply:
Yeah... we could keep it in camel case but this what it actually is in the hail table/vcf.

Reply:
The casing is all over the map with these metrics

Reply:
I'm going to leave it as `inbreeding_coeff` because 

* The browser is setup to display the QC metric names as they are in the data. Changing this would require adding renaming logic
* The other metrics are a mix of camel case, upper snake case, and upper/lower mixed snake case. Since there is  no clear existing convention, it doesn't seem important to enforce one now
* People who see this field in the browser will have an easier time finding it in the hail table if they care to